x-pack/plugins/security/server/lib/authorization/privileges/privileges.test.ts

```diff
@@ -0,0 +1,631 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
```

Comment:
This `ts-ignore` is unnecessary, since `xpack_main/types` is a TypeScript file. It complains because the import path is incorrect